examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic surface plotting with plot_surf_stat_map
+==========================================================
+
+See :ref:`plotting` for more details.
+"""
+import matplotlib.pyplot as plt
+from nilearn import plotting
+from nilearn import datasets
+
+# Retrieve the data
```

Comment:
All examples start with ############### to have head line representation. If you want you can have a look at other examples.